File: LeetCode/1470.Shuffle_The_Array/main.py
#!/usr/bin/python

from solution import Solution

# Unit Test
nums1 = [2,5,1,3,4,7]
n = 3
s = Solution()
if (s.shuffle(nums1, n)==[2,3,5,4,1,7]):
    print("Pass")
else:
    print("Test 1 Failed")

File: LeetCode/1470.Shuffle_The_Array/main.cpp
#include "solution.h"

// Unit Test

int main(){
    Solution s;
    vector<int> v1 {2,5,1,3,4,7};
    int n1 = 3;
    vector<int> v2 {1,2,3,4,4,3,2,1};
    int n2 = 4;
    vector<int> v3 {1,1,2,2};
    int n3 = 2;
    vector<int> res1 = s.shuffle(v1, n1);
    vector<int> res2 = s.shuffle(v2, n2);
    vector<int> res3 = s.shuffle(v3, n3);
    
    return 0;
}